Find a functionally equivalent code:android.media.AudioManager.querySoundEffectsEnabled:COMMENT
Method Modifier: private     
Comment:/**
 * Settings has an in memory cache, so this is fast.
 */

@@ -1,3 +1,3 @@
 {
-    return Settings.System.getInt(mContext.getContentResolver(), Settings.System.SOUND_EFFECTS_ENABLED, 0) != 0;
+    return Settings.System.getIntForUser(mContext.getContentResolver(), Settings.System.SOUND_EFFECTS_ENABLED, 0, user) != 0;
 }

